client/src/components/PageComponents/Header/Header.tsx

```diff
@@ -8,21 +9,67 @@ import {
 } from "./Header.css";
 
 const Header = () => {
+  const navigate = useNavigate();
+
+  const gotoMainPage = () => {
+    navigate("/");
+  };
+
+  const gotoSchedulePage = () => {
+    navigate("/schedule");
+  };
```

Comment:
라우팅을 다룰때 라우팅 경로는 상수로 다루는게 좋습니다~
```suggestion
  const gotoSchedulePage = () => {
    navigate(ROUTES.SCHEDULE);
  };
```